Makefile

```diff
@@ -3,6 +3,7 @@ SOURCES := $(shell find $(SOURCEDIR) -name '*.go')
 ROOT := $(shell pwd)
 LOCAL_ENI_PLUGIN_BINARY=bin/plugins/eni
 LOCAL_IPAM_PLUGIN_BINARY=bin/plugins/ipam
+LOCAL_BRIDGE_PLUGIN_BINARY=bin/plugins/ecs-bridge
 GIT_PORCELAIN=$(shell git status --porcelain | wc -l)
 GIT_SHORT_HASH=$(shell git rev-parse --short HEAD)
 VERSION=$(shell cat $(ROOT)/VERSION)
```

Comment:
All the plugins are sharing the same version, I think each plugin should have its own version, right?